modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1086,6 +1113,18 @@ class Edit_User extends \NDB_Form
                 'Radiologist' and 'Pending Approval').";
             }
         }
+
+        //======================================
+	//	Validate Active time Windows (from - to)
+	//
+	//	Note: the only validation included for the moment
+	//	is that the time form which the user is active 
+	//	should be less or equal to the time to it will be active. 
+        //======================================
+
+	if(($values['active_to'] != NULL) && ($values['active_from'] > $values['active_to'])){
+		$errors['active_timeWindows'] = "Please notice that the <i>\"Active from\"</i> date should be lesser or equal to the <i>\"Active to\"</i> date.";
```

Comment:
We usually don't inject HTML directly into error messages like this as a matter of policy. If the italics are very important to you here, this should be done in the Smarty template instead.